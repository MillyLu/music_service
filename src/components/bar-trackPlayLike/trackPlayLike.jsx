import {  useContext} from 'react';
import { ThemeContext } from '../../ThemeProvider';
import * as Styled from './styles';

export function TrackPlayLike({handleAddFavorites}){
    const {theme} = useContext(ThemeContext);
    return(
        <Styled.TrackPlayLike>
            <Styled.TrackPlayLikeSvg data-testid="like" alt='like' theme={theme} onClick={handleAddFavorites}>
            </Styled.TrackPlayLikeSvg>
        </Styled.TrackPlayLike>
    )
}