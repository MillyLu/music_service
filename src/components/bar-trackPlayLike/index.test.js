/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
// import { ShallowRenderer, createRenderer } from 'react-shallow-renderer';
import { TrackPlayLike } from './trackPlayLike';



describe('<TrackPlayLike/>', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<TrackPlayLike />);
            const result = renderer.getRenderOutput();
            expect(result).toMatchSnapshot();
          })
      });