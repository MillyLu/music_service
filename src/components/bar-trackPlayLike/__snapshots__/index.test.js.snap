// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackPlayLike/> should render successfully and match snapshot 1`] = `
<styled.div>
  <Styled(Component)
    alt="like"
    data-testid="like"
    theme={0}
  />
</styled.div>
`;
